hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -358,6 +358,13 @@ public static String createNewFileIdPfx() {
     return UUID.randomUUID().toString();
   }
 
+  /**
+   * Returns prefix for a file group from fileId.
+   */
+  public static String getFileIdPfxFromFileId(String fileId) {
```

Comment:
This looks dangerous. Instead, can you split by delimiters and take the right part.

Reply:
done